example.js

```diff
@@ -0,0 +1,43 @@
+// First let's import the logging module.
+var logging = require('./lib/log');
+
+// Now, let's create a module-level object.  The name
+// parameter is mandatory and should be the name of your
+// app.  The object passed in here is just a bunyan log
+// config object.
+var log = logging("my-example", {module: "main"});
```

Comment:
Okay, so I'll only instantiate `log = logging(....)` once in my application correct? And then pass it around...

I think this is a good idea... it won't be ugly when we use the component loader - I think this is a good idea...